gradle/testing/defaults-tests.gradle

```diff
@@ -47,7 +47,7 @@ allprojects {
            description: "Number of forked test JVMs"],
           [propName: 'tests.haltonfailure', value: true, description: "Halt processing on test failure."],
           [propName: 'tests.jvmargs',
-           value: { -> propertyOrEnvOrDefault("tests.jvmargs", "TEST_JVM_ARGS", "-XX:TieredStopAtLevel=1") },
+           value: { -> propertyOrEnvOrDefault("tests.jvmargs", "TEST_JVM_ARGS", "-XX:TieredStopAtLevel=1 -XX:ReservedCodeCacheSize=120m") },
```

Comment:
`-XX:ReservedCodeCacheSize=120m`

 Is `120m` just a guess at this point?

Can we add a comment about why? Maybe a "the JDK sets this to XYZm and we need at least 120m".

Reply:
Yea, it's just a guess. I added a comment which is mostly a copy from https://docs.oracle.com/en/java/javase/17/docs/specs/man/java.html#advanced-jit-compiler-options-for-java